pkg/cvo/cvo.go

```diff
@@ -224,7 +226,7 @@ func (optr *Operator) InitializeFromPayload(restConfig *rest.Config, burstRestCo
 	optr.releaseVersion = update.ImageRef.Name
 
 	// attempt to load a verifier as defined in the payload
-	verifier, err := verify.LoadFromPayload(update)
+	verifier, err := verify.LoadFromPayload(update, optr)
```

Comment:
instead of passing the whole optr object, can we pass a small type that just does HTTPClient .. 
you can `optr.verifyClientBuilder` or some such.

Reply:
I think we can do that if we store pointers to the listers for proxy and trust bundle in the new type.

Reply:
Created a wrapper struct to hold the function.